README.MD

```diff
@@ -1,19 +1,33 @@
 # UW Pub Sub
 
+Node Version:
+
+`>= v7.9.0`
+
 Usage:
 
-`yarn add git+ssh://git@github.com/utilitywarehouse/uw-lib-pubsub.js`
+`yarn add git+ssh://git@github.com/utilitywarehouse/uw-lib-pubsub.js no-kafka`
+
+Testing:
+
+`npm test`
 
 wiring.yml:
 
 ```
+kafka.producer:
+    class: 'no-kafka::Producer'
+    with:
+      -
+        connectionString: '@kafka.brokers'
```

Comment:
might be worth showing how this looks like, i.e a comma separated string for topics `kafka.brokers: 'topic1,topic2,topic3`